shop-be/import-service/src/functions/importProductsFile/handler.ts

```diff
@@ -0,0 +1,43 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/apiGateway';
+import { middyfy } from '@libs/lambda';
+import { PutObjectCommand, PutObjectCommandInput, S3 } from '@aws-sdk/client-s3';
+import { getSignedUrl } from '@aws-sdk/s3-request-presigner';
+import { PREFIX, REGION } from '@libs/constants';
+
+const importProductsFile: ValidatedEventAPIGatewayProxyEvent = async (event) => {
+  const { queryStringParameters: { name } } = event;
+
+  if (!name) {
+    return {
+      statusCode: 400,
+      headers: { 'Access-Control-Allow-Origin': '*' },
+      body: "No file name provided"
+    }
+  }
+
+  const s3 = new S3({ region: REGION });
```

Comment:
I've already seen this line - work with s3 is a good candidate for the service

Reply:
moved